librz/debug/p/debug_dmp.c

```diff
@@ -0,0 +1,467 @@
+// SPDX-FileCopyrightText: 2021 GustavoLCR <gugulcr@gmail.com>
+// SPDX-License-Identifier: LGPL-3.0-only
+
+#include <rz_core.h>
+#include <rz_debug.h>
+#include <dmp_specs.h>
+#include <dmp64.h>
+#include <pe_specs.h>
+#include <winkd.h>
+
+// rz_type_db_struct_member_offset() does packed offsets only, we want aligned
```

Comment:
We should open an general issue to track this and possibly implement in the future.